src/lib/ledger/provideContractData.js

```diff
@@ -54,6 +54,8 @@ module.exports = async (dvf, transport, tokenAddress = '', transferQuantization)
     if (transferQuantization) {
       await _transport.starkProvideQuantum_v2(transferTokenAddress, tokenAddress ? 'erc20' : 'eth', transferQuantization, null)
     }
+  } catch (e) {
+    return { unsafeSign: true }
```

Comment:
Might be worth a `console.warn` here (unless we think it would pollute too much the console)